pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractDispatcherMultipleConsumers.java

```diff
@@ -64,6 +64,10 @@ public synchronized boolean canUnsubscribe(Consumer consumer) {
         return consumerList.size() == 1 && consumerSet.contains(consumer);
     }
 
+    public boolean isClosed() {
+        return IS_CLOSED_UPDATER.get(this) == TRUE;
```

Comment:
nit: for regular get/set operations, we can just use the variable itself `return isClosed == TRUE`

Reply:
Fixed.